helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -414,8 +414,12 @@ receivers:
       # Clean up log record
       - type: move
         id: clean-up-log-record
+        output: clean-up-time-attribute
         from: attributes.log
         to: body
+      - type: remove
+        id: clean-up-time-attribute
+        field: attributes.time
```

Comment:
No need to add id unless it's being used as a reference in a non-sequential flow

```suggestion
        from: attributes.log
        to: body
      - type: remove
        field: attributes.time
```